controller/node_controller.go

```diff
@@ -1402,7 +1405,7 @@ func (nc *NodeController) deleteOrphans(node *longhorn.Node, diskName string, di
 	return nil
 }
 
-func (nc *NodeController) createOrphans(node *longhorn.Node, diskName string, diskInfo *monitor.CollectedDiskInfo, newOrphanedReplicaDataStores map[string]string) error {
+func (nc *NodeController) createDataStoreOrphans(node *longhorn.Node, diskName string, diskInfo *monitor.CollectedDiskInfo, newOrphanedReplicaDataStores map[string]string) error {
```

Comment:
Can you elaborate more on why we rename to `createDataStoreOrphans`? What does DataStore mean in this context?

Reply:
In this PR, besides the existing data store orphan, we introduce new types of orphans for runtime instance. The node controller observes the disk status change, and has no knowledge of engine/replica instances. Rename the method here to emphasize that the node controller will create only the data store orphans.

Reply:
I think we should use `createOrphans` and `createOrphan` that are similar to `createVolume` and so on. It is a convention and developers already know it is for resource creation.